client/flutter/assets/svg/logo_banner.svg

```diff
@@ -0,0 +1,40 @@
+<svg width="581" height="178" viewBox="0 0 581 178" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
WHO identity assets should go in the `identity` dir and then be symlinked to (as it under a separate license)

We probably should add that to the CONTRIBUTING doc at some point..

Reply:
symlinks aren't a universal standard i.e. they don't work on windows.
Could we add a readme.md to the `identity` folder with clickable links to folders within the flutter `assets` dir?

Reply:
@kassim thanks for being very circumspect on this! We actually changed our license to broadly cover all WHO identity assets without needing them to be in the `identity` folder, so we should be ok here. Unblocking